package.json

```diff
@@ -70,7 +70,7 @@
     "typescript": "^4.9.5"
   },
   "dependencies": {
-    "@openzeppelin/defender-sdk": "^1.2.0",
+    "@openzeppelin/defender-sdk": "^1.0.0",
```

Comment:
Shouldn't this be 1.5.0?

Reply:
Yep it automatically takes the latest minor so we dont have to keep updating this atleast until 1.99.0